conftest.py

```diff
@@ -31,3 +35,19 @@ def pytest_addoption(parser):
     parser.addoption(
         "--noskip", action="store_true", default=False, help="Treat skipped tests as errors"
     )
+
+
+def pytest_runtestloop():
+    # These files are so big they exceed our "immutable symlink" hueristic. However, our integration
+    # tests run Pants in the sandbox, and Pants doesn't allow globbing absolute symlinks.
+    # Therefore we re-materialize them from their symlink.
```

Comment:
We should probably have this point to a ticket that discusses whether/how to expose an override for this behavior to users (only for `test`? in general?). I expect that we will encounter more issues like this during the alpha release.

Reply:
I suspect this issue really only matters in very niche cases, but I could be wrong. Here, we're running Pants in a Pants sandbox, and that's already a bit of a stretch :stuck_out_tongue: But again, I could be wrong...